tag_database

```diff
@@ -90,6 +90,7 @@ flattenObject:object,recursion,intermediate
 flip:adapter,function,intermediate
 forEachRight:array,function,intermediate
 formatDuration:date,math,string,utility,intermediate
+formToObject:browser,utility,object
```

Comment:
There is no expertise level - add `,beginner` at the end of the line.

Reply:
ok, i fix it